feature-libs/checkout/base/components/checkout-delivery-address/checkout-delivery-address.component.ts

```diff
@@ -329,4 +335,12 @@ export class CheckoutDeliveryAddressComponent implements OnInit {
   protected shouldUseAddressSavedInCart(): boolean {
     return !!this.checkoutConfigService?.shouldUseAddressSavedInCart();
   }
+
+  private getCardRole(isCardSelected: boolean): 'button' | 'region' {
```

Comment:
Nice use of the feature flags here! Lets make this method `protected` instead of `private` so that lib users can still access it if they have a need to see/modify it without cloning the repo.

Reply:
Lets also add a unit test assertion to check that the `role` is set correctly since there is some logic here.